index.html

```diff
@@ -1592,17 +1590,40 @@ <h3 id="applying">
         <div class="issue" data-number="446"></div>
       </section>
     </section>
-    <section>
+    <section data-dfn-for="webappmanifest" data-link-for="webappmanifest">
       <h2>
         Manifest and its members
       </h2>
       <p>
         A <dfn>manifest</dfn> is a JSON document that contains startup
         parameters and application defaults for when a web application is
-        launched. A manifest consists of a top-level <a>object</a> that
-        contains zero or more members. Each of the members are defined below,
-        as well as how their values are processed.
+        launched. A manifest consists of a top-level <dfn>WebAppManifest</dfn>
+        <a>object</a> that contains zero or more members.
+        Each of the members are defined below, as well as how their values are processed.
       </p>
+      <pre class="idl">
+          [NoInterfaceObject]
```

Comment:
Remove [NoInterfaceObject]  ... it's not an interface, it's a dictionary 😉. Also, you should never ever need to use NoInterfaceObject in a spec unless you are creating some kind of mix-in.

Reply:
(I'll just emphasize, if you ever see `NoInterfaceObject` in a spec, yell at them... or tell me, and I will yell at them 📣😝)

Reply:
Sure :-) done...